assets/quantity-popover.css

```diff
@@ -22,12 +23,19 @@ quantity-popover volume-pricing li {
   max-width: 36rem;
 }
 
+.quantity-popover__info .button-close,
+.variant-remove-total quick-order-list-remove-all-button,
+.quick-order-list-total__confirmation quick-order-list-remove-all-button {
+  --shadow-opacity: 0;
```

Comment:
We'll need to prevent borders also. If we apply the [exact settings shown in the issue](https://github.com/Shopify/dawn/issues/2845#issue-1805570855), the borders will be visible.

Also for a11y, I noticed that `quick-order-list-remove-all-button` is not focusable via keyboard navigation.